packages/block/src/block.ts

```diff
@@ -69,13 +71,13 @@ export class Block {
     // parse transactions
     let transactions = []
     for (const txData of txsData) {
-      transactions.push(Transaction.fromRlpSerializedTx(txData, opts as TxOptions))
+      transactions.push(Transaction.fromValuesArray(txData as Buffer[], opts as TxOptions))
```

Comment:
Don't know why TypeScript doesn't auto-cast this as a Buffer[], it should do that, but it fails to compile for me if I don't add this `as Buffer[]`.

Reply:
that's okay, i think ev and i have also encountered this in a select few other places and we did the same 👍